controllers/components/datasciencepipelines/datasciencepipelines_support.go

```diff
@@ -40,6 +40,7 @@ var (
 		"V2_DRIVER_IMAGE":                  "RELATED_IMAGE_ODH_ML_PIPELINES_DRIVER_IMAGE",
 		"V2_LAUNCHER_IMAGE":                "RELATED_IMAGE_ODH_ML_PIPELINES_LAUNCHER_IMAGE",
 		"IMAGESV2_ARGO_MLMDGRPC":           "RELATED_IMAGE_ODH_MLMD_GRPC_SERVER_IMAGE",
+		"IMAGES_PIPELINES_RUNTIME_GENERIC": "RELATED_IMAGE_ODH_ML_PIPELINES_RUNTIME_GENERIC",
```

Comment:
probably it will eneded like `RELATED_IMAGE_ODH_ML_PIPELINES_RUNTIME_GENERIC_IMAGE`